app/javascript/src/components/Dashboard/Notes/List/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
The List folder only has one file. Which is suggesting to us that we don't need a folder for it in the fist place. Thus keep this logic within `Notes/List.jsx`